Engine/CMakeLists.txt

```diff
@@ -419,6 +419,8 @@ target_sources(engine
     media/video/video.h
     platform/base/agsplatformdriver.cpp
     platform/base/agsplatformdriver.h
+    platform/base/agsplatform_unix.cpp
+    platform/base/agsplatform_unix.h
     platform/base/sys_main.cpp
```

Comment:
the cpp file would need to be added to Engine/Makefile too i guess ?

Reply:
so, the base platform apparently has some magic that makes unnecessary, because platform/base is already added.

https://github.com/adventuregamestudio/ags/blob/811f6c126fcff1cc5a0db8b6e0a1b9fca2550235/Engine/Makefile-objs#L30

but check the commit below, that one would be needed if there was a bsd makefile (I think)